src/modules/dashboard/components/Dialogs/EditExpenditureDialog/ChangedMultiple/ChangeItem/ChangeItem.tsx

```diff
@@ -0,0 +1,57 @@
+import { nanoid } from 'nanoid';
+import React, { useCallback } from 'react';
+
+import { FormSection } from '~core/Fields';
+import Icon from '~core/Icon';
+import { Colony } from '~data/index';
+
+import NewDelay from '../../NewDelay';
+import NewRecipient from '../../NewRecipient';
+import NewValue from '../../NewValue';
+
+import styles from './ChangeItem.css';
+
+const displayName =
+  'dashboard.EditExpenditureDialog.ChangedMultiple.ChangeItem';
+
+interface Props {
+  newValue: any;
+  oldValue?: any;
```

Comment:
Why are these 2 values any? Usually, we try to avoid it. Or at least make a comment as to why it has to be any.